containers/codespaces-linux/.devcontainer/library-scripts/python-ml-debian.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
An exact copy of this will also need to go in the root [script-library](https://github.com/microsoft/vscode-dev-containers/tree/main/script-library).  At release-time, the copy in each individual `containers/` folder is synced with the "original" copy in the root folder.